README.rst

```diff
@@ -18,6 +18,8 @@ The dependencies are available via pip::
    sudo apt install python3-pip
    # For RPM-based distros
    sudo yum install python3-pip
+   # For OpenSUSE
```

Comment:
distro name is openSUSE not OpenSUSE

Reply:
look opensuse.org

Reply:
https://github.com/Zaryob/openzfs-docs/pull/1 I created pull request. I fixed other typo issues. Please merge 😁

Reply:
Ok. Im gonna review and merge.